sernet.gs.ui.rcp.main/src/sernet/gs/ui/rcp/hibernate/CnATreeElement.hbm.xml

```diff
@@ -121,6 +121,15 @@
         </set>
         
         <property name="iconPath" column="icon" type="string" length="256"/>
+        
+        <property name="templateTypeValue" type="string">
```

Comment:
After migrating an old DB to the new version I get this exception:

`java.lang.IllegalArgumentException: No enum constant sernet.verinice.model.common.CnATreeElement.TemplateType.none
	at java.lang.Enum.valueOf(Enum.java:238)
	at sernet.verinice.model.common.CnATreeElement$TemplateType.valueOf(CnATreeElement.java:1)
	at sernet.verinice.model.common.CnATreeElement.getTemplateType(CnATreeElement.java:877)
	at sernet.verinice.model.common.CnATreeElement.isImplementation(CnATreeElement.java:930)
	at sernet.gs.ui.rcp.main.bsi.views.TreeElementDecoratingLabelProvider.getForegroundColorForElement(TreeElementDecoratingLabelProvider.java:73)`

Change the default value to `default="'NONE'"`